test/mlir/conversion/onnx_to_torch/maxpool/asymetricmaxpool.onnx.mlir

```diff
@@ -0,0 +1,14 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", llvm.target_triple = "x86_64-unknown-linux-gnu"}  {
+  func @main_graph(%arg0: tensor<1x5x10x10xf32>) -> tensor<1x5x5x5xf32> attributes {input_names = ["0"], output_names = ["2"]} {
+//CHECK: [[STRIDE:%.]] = torch.prim.ListConstruct %int2{{_*[0-9]*}}, %int1{{_*[0-9]*}} :
+//CHECK: [[DILATION:%.]] = torch.prim.ListConstruct %int1, %int1 :
+//CHECK: [[PAD:%.]] = torch.prim.ListConstruct %int0, %int0{{_*[0-9]*}} :
+//CHECK: [[BIAS:%.]] = torch.prim.ListConstruct %int3, %int2 :
+//CHECK: torch.aten.max_pool2d %arg0, [[BIAS]], [[STRIDE]], [[PAD]], [[DILATION]], %false :
```

Comment:
One `CHECK` but two `onnx.MaxPoolSingleOut` ops,

Reply:
Liam, 

Even here we have removed the additional MaxPoolSingleout which was not required.